### PR TITLE
Feat/013 backend resolve endpoint

### PR DESCRIPTION
**Pull request by EliasSalom**

### CODE REVIEW

File: apps/client/API/academic/postAcademic.ts [hunk 1]
```diff
@@ -1,11 +1,17 @@
 import { backendInstance } from '../api';
-import { useQuery } from '@tanstack/react-query';
+import { useMutation, useQueryClient } from '@tanstack/react-query';
 import { AcademicCreated, IAcademic } from './types';
 
 const postAcademic = async (data: AcademicCreated): Promise<IAcademic> => {
-  return await backendInstance.post('/users/register', data);
+  console.log(data);
```
Comment (HassanHassouna) [inline apps/client/API/academic/postAcademic.ts]: Please remove logs when you pushed your code

File: apps/client/components/Dialog/dialogLogic.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import { AcademicCreated, AcademicField } from '../../API/academic/types';
+
+export const AcademicDialogLogic = () => {
+  const [academicDetail, setAcademicDetail] = useState<AcademicCreated>();
+  const [isOpen, setIsOpen] = useState(false);
+  const { mutate } = usePostAcademicMutation();
+  const onClose = () => setIsOpen(false);
+  const onOpen = () => setIsOpen(true);
+  const OnSubmit = async () => {
+    console.log(academicDetail);
```
Comment (HassanHassouna) [inline apps/client/components/Dialog/dialogLogic.ts]: Same here

**Review by HassanHassouna — CHANGES_REQUESTED** [review]

File: apps/client/API/academic/postAcademic.ts (hunk 1 above)
Comment (EliasSalom) [reply apps/client/API/academic/postAcademic.ts]: done, the `console.log()` was removed

File: apps/client/components/Dialog/dialogLogic.ts (hunk 2 above)
Comment (EliasSalom) [reply apps/client/components/Dialog/dialogLogic.ts]: done, the `console.log()` was removed

**Comment by EliasSalom** [comment]
all the comment was resolved

**Pull request merged**